src/aiida_quantumespresso/parsers/pw.py

```diff
@@ -274,8 +274,9 @@ def is_ionically_converged(self, trajectory, except_final_scf=False):
 
         if 'settings' in self.node.inputs:
             settings = _uppercase_dict(self.node.inputs.settings.get_dict(), dict_name='settings')
-
-        fixed_coords = settings.get('FIXED_COORDS', None)
+            fixed_coords = settings.get('FIXED_COORDS', None)
```

Comment:
Maybe we can change the code slightly, so it's more uniform with other places where we deal with `settings`, i.e.:

https://github.com/aiidateam/aiida-quantumespresso/blob/d9b18a7c20ce023018755c202f8d06cbf8bd27c5/src/aiida_quantumespresso/calculations/__init__.py#L227-L231

Reply:
Hi @mbercx , sure your modification is perfect. I did not check when/how this changed but I guess this was tested on the QE app and there probably there is always some settings in ions. I had to use our nanoribbons app, for the first time after we migrated it to aiida-2.x , and there I noticed the problem.